src/issueFinder.ts

```diff
@@ -1,4 +1,3 @@
-import { GitHubIssueFetcher, GitHubIssue } from './gitHubIssueFetcher';
 import * as mariner from './mariner/index'; // This is used during development
```

Comment:
Anything inside src/ can refer to other stuff in src/ without problems. We should get rid of the import * as mariner line, and just have anything it needs get imported from ./ like the line that was removed.